src/NRules/NRules/Aggregators/MultiKeySortedAggregator.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using NRules.RuleModel;
+using NRules.Utilities;
+
+namespace NRules.Aggregators
+{
+    /// <summary>
+    /// A key selecting expression and direction used for sorting
+    /// </summary>
+    internal class SortCriteria
```

Comment:
BTW, you don't need XML documentation on internal classes

Reply:
Fixed. I wasn't sure where to draw the line as the Aggregator and AggregatorFactory classes do have XML documentation.

Reply:
The idea is to supply XML doc for public types, since I'm generating API documentation from it. Internal stuff mostly uses no comments approach, except where it is helpful.